docs/parser/flexer.md

```diff
@@ -88,6 +88,32 @@ they mostly choose the _longest_ match instead. Once the pattern is matched, the
 associated code is executed and the process starts over again until the input
 stream has been consumed.
 
+### Automatic code generation
+
+In order to avoid getting lexer definition out of sync with its implementation
```

Comment:
```suggestion
### Automatic Code Generation
In order to avoid getting lexer definition out of sync with its implementation
```

You also need to update the table of contents to reflect the new section that you've added.